---
output: 
  word_document:
    pandoc_args: [ "--csl", "american-journal-of-botany.csl"]
    toc: no
    reference_docx: american-journal-of-botany.docx
bibliography: references.yaml
---
<br>
<br>
<br>

Nitta and Chambers---DNA barcoding of fern gametophytes

# Identifying cryptic fern gametophytes using DNA barcoding: A review

Joel H. Nitta^1^ and Sally M. Chambers^2^

^1,3^Department of Biological Sciences, Graduate School of Science, The University of Tokyo, 2-11-16 Yayoi, Bunkyo-ku, Tokyo, 113-0032, Japan

^2^Marie Selby Botanical Gardens, Botany Department, Sarasota FL 34236 USA

^3^Author for correspondence: joelnitta@gmail.com

Manuscript received _______; revision accepted _______.

Number of words:

Ferns and lycophytes are unique among land plants for having sporophyte (diploid) and gametophyte (haploid) generations that can grow independently of each other. 
While most studies of fern ecology focus on the more obvious sporophytic stage, the gametophyte is critically important, as it is the sexual phase of the life cycle. 
Yet, fern gametophytes have long been neglected in field studies due to their small size and cryptic morphology. 
DNA barcoding is a powerful method that can be used to identify field-collected gametophytes to species and allow for detailed study of their ecology. 
Here, we review the state of DNA barcoding as applied to fern gametophytes. 
First, we trace the history of DNA barcoding and how it has come to be applied to fern gametophytes. 
Next, we summarize case studies that show how DNA barcoding has been used to better understand fern species distributions, gametophyte ecology, and community ecology. 
Finally, we propose avenues for future research using this powerful tool, including next-generation DNA sequencing for in-field identification of cryptic gametophytes.

Key words: DNA barcoding; fern; gametophyte; MinIon; rbcL

**INTRODUCTION**

Ferns and lycophytes (collectively known as pteridophytes) are unique among land plants for having sporophyte (diploid) and gametophyte (haploid) life cycle stages that can grow independently of each other, and therefore may occur separately in nature. 
This is not the case for other the major land plant groups: in bryophytes (mosses, liverworts, and hornworts), the sporophyte is nutritionally dependent upon and physically attached to the gametophyte, whereas in seed plants, the gametophyte is either contained within the flower (female gametophyte), or does not grow into a nutritionally independent organism (male gametophyte; pollen). 
In these plants, one can sample the dominant life stage without regards for the other, dependent life stage. 
However, for pteridophytes, sampling one life stage alone is not sufficient to understand the complete distribution of the organism. 
This has critical implications for understanding the biology of ferns and lycophytes. 

The vast majority of ecological studies of pteridophytes has sampled only sporophytes, since this is the larger, more obvious life stage. 
The gametophytes of pteridophytes, however, are much smaller (typically < 1 cm) and morphologically cryptic; i.e., they typically lack morphological characters that can be used to differentiate species. 
As the sexual stage (producing gametes and undergoing fertilization) gametophytes are critically important for pteridophyte biology. 
Gametophytes have long been cultivated from spores in the lab to gain insight into their morphology, development, and reproduction. 
However, lab studies are inevitably limited with regards to understanding the mechanisms underlying population dynamics in the field. 
For this, we must either sample gametophytes directly or make inferences from their larger diploid counterparts.

DNA barcoding, the use of a standard, short region of DNA for species identification [@Hebert2003b], is a powerful method to enable species identification of field-collected fern gametophytes. 
Here, we review the use of DNA barcoding as applied to fern gametophytes, starting with the development of the method, then discussing various case studies, and concluding with future directions. 
Although the concepts discussed here all apply equally to ferns and lycophytes, we are not aware of any studies using DNA barcoding of lycophytes, so our discussion will focus on ferns.

**DISCUSSION**

**Beginnings: Proof-of-concept studies**

The earliest examples of DNA barcoding applied to gametophytes were proof-of-concept studies that demonstrated the viability of the method. 
The basic approach taken by these studies is still used today: Sanger sequencing of one or more of the most frequently sequenced plastid genes in ferns. 
@Schneider2006a were the first to do so, using *rbcL* to identify a gametophyte individual that had been in cultivation for > 30 years as *Osmunda regalis*. 
Likewise, @Li2009a identified a single gametophyte individual kept in cultivation as a species of *Lomariopsis*. 
@Li2009a expanded gene sampling relative to @Schneider2006a to include more rapidly evolving intergenic regions in addition to the widely sequenced, but relatively more slowly evolving, *rbcL*. 
Despite this, they could not determine the species due to lack of an exact match in reference sequences from GenBank. 
These two studies highlight some of the challenges facing DNA barcoding: 1. choice of barcode marker and 2. development of an appropriate reference library.

**Choosing a barcode marker**

The power of DNA barcoding lies in its universality: the ability to sequence a standard region of DNA and use this for identification of (theoretically) any unknown organism. 
The choice of mitochondrial *COI* was as a universal barcode marker for animals was quickly adopted because this marker can be reliably sequenced across a broad phylogenetic range and tends to show high discrimination power at the species level [@Hebert2003a; @Hebert2003b]. 
However, the quest for a similar universal barcode marker in plants has proven to be much more difficult [@Hollingsworth2011]. 
As a practical matter, *rbcL* was already the most commonly sequenced marker in plants, and an obvious strong candidate. 
However, *rbcL* has a relatively slow rate of evolution at the species level (particularly in angiosperms), and cannot be used to distinguish between species in many cases. 
From the period spanning the establishment of the Consortium of the Barcode of Life (CBOL) project in 2004 to the early 2010s, a series of studies debated various combinations of candidate barcodes to use across all plants [@Chase2005; @Kress2005; @Newmaster2006; @Chase2007; @Kress2007a; @Taberlet2007; @Lahaye2008; @Chen2010; @Hollingsworth2011a]. 
A two-barcode system was eventually established based on *rbcL* and *matK* [@Group2008], but due to a rearrangement of the chloroplast in the ancestor of ferns, no universal primers are available that work for *matK* across all ferns [@Li2011], and studies continued within ferns to find a set of markers that would work for this group of plants.

@Ebihara2010 tested three regions based on sampling the fern flora of Japan, and suggested *rbcL* and the *trnH-psbA* intergenic spacer. 
@Li2011 showed that *trnH-psbA* is a poor choice due to its decelerated rate of evolution in most groups of ferns [@Li2016b]. 
Rather, @Li2011 developed additional primers for *matK* and suggested this region be used in addition to *rbcL*. 
Another region that has received strong support due to its high rates of evolution is the *trnLF* spacer. 
This was tested in the ferns of Europe [@DeGroot2011a] and used to identify vittarioid gametophytes in Taiwan [@Chen2013a]. 
Recently, [@Nitta2020c] compared rates of interspecific variation in *rbcL* in three fern floras (French Polynesia, Costa Rica, and Japan), and found that this marker had a higher identification failure rate in continental areas (Costa Rica and Japan) relative to the isolated islands of French Polynesia, which they attributed to the possibilty of higher rates of *in-situ* speciation in continental regions vs. long-distance dispersal in remote oceanic islands. 
This highlights the need to consider biogeographical context in DNA barcoding of fern gametophytes, at least when the provenance of specimens is known. 

Although @Li2011 demonstrated that *matK* is a strong candidate as a second DNA barcode marker in ferns, their recommendation was predicated on the development of additional primers for this region, but in the decade since, there has been no development of universal *matK* primers in ferns. 
Rather, most studies have taken an ad-hoc approach, sampling genes that make sense for the purpose of each study (while typically including *rbcL*). 
We thus consider DNA barcoding in ferns in the broad sense (use of DNA sequences to identify cryptic gametophytes), and do not limit our discussion to any particular "official" barcode.

**Range expansions and independent gametophytes**

Following the initial demonstrations by @Schneider2006a and @Li2009a that DNA barcodes could be used to identify cultivated fern gametophytes and the testing of various combinations of barcode regions, a major use of DNA barcodes in ferns has been documenting the distribution of gametophytes growing at large distances from their sporophytic counterparts [reviewed in @Pinson2016a]. 
The ability of fern gametophytes to grow spatially separated from sporophytes reaches an extreme in the so-called "independent gametophytes", species that are known from the gametophyte stage only and no longer produce sporophytes. 
These were first documented in temperate Eastern North American forests in otherwise typically tropical lineages, including vittarioid ferns, filmy ferns, and grammitid ferns [@Farrar1967]. 
The gametophytes of these tropical lineages tend to have distinct morphologies that allowed them to be distinguished at least to genus or similar higher taxonomic group previous to the development of DNA barcoding [@Pinson2016a].

The knowledge of these independent gametophyte populations in the Eastern United States made them a natural target for study when DNA barcoding became available.
@Ebihara2008a used *rbcL* and *rps4-trnS* sequences to identify the independent gametophyte of *Trichomanes* (*s.l.*) *intricatum* Farrar from the Eastern U.S. as being very closely related to *Crepidomanes schmidtianum* (Zenker ex Tasch.) K. Iwats. from Eastern Asia. 
Although *T. intricatum* was known to belong to the predominantly tropical Hymenophyllaceae, prior to DNA sequencing, there were no suggestions that it might be most closely related to an East-Asian species. 
Similarly, @Pinson2016 utilized four plastid (*atpA*, *chlN*, *rbcL*, and *rpoA*) and one nuclear gene (*DET1*) to investigate the identity of another of the independent gametophytes described by Farrar, *Vittaria appalachiana* Mickel and Farrar, and concluded it was nested *Vittaria graminifolia* Kaulf., from Central and South America. 
Another successful identification of North American independent gametophytes was on the Pacific North Coast, where DNA barcoding revealed that *Hymenophyllum wrightii* Bosch, a primarily East Asian species only occurring rarely as a sporophyte in North America, actually has abundant populations of independent gametophytes [@Duffy2015].

The East Asian origins of *T. intricatum* and *H. wrightii* spurred on the search for additional populations of independent gametophytes in East Asia.
@Lee2020 recently documented additional populations of *T. intricatum* and *H. wrightii* in Korea. 
Furthermore, multiple independent vittarioid gametophytes have been found in Taiwan [@Kuo2017b], Korea [@Park2020; @Park2021], and Japan [@Ebihara2013; @Kuo2017b]. 
The careful studies of @Kuo2017b and @Park2021 including observations of young sporophytes have helped to provide insights into the mechanisms preventing sporophyte production in these populations.

<< Question about @Pinson2017: the sporophytes are *V. graminifolia*, which is a match for *V. appalachiana*. 
How do you know the Broxton Rocks *V. graminifolia* are not sporos of *V. appalachiana*?  >>

* H. wrightii in Alaska [@Duffy2015]
* Independent vittarioid gametophytes in S. Japan and Taiwan [@Kuo2017b]
* Vittaria graminifolia in GA [@Pinson2017]
* Vittarioid ferns on Jeju island [@Park2020; @Park2021]
* P. makinoi, C. intricatum, H. wrightii in Korea [@Lee2020]

**Scaling up: Community ecology of fern gametophytes**

Beyond identifying individual populations of gametophytes, some DNA barcoding studies have sought to compare community-level diversity between gametophytes and sporophytes. 
Since gametophytes have the ability to occur over wider ranges and only produce sporophytes within a portion of that range, it is possible that community structure may vary between the two life stages. 
Although DNA barcoding is a powerful tool to test this hypothesis, it has only been utilized a small number of times. 

@Ebihara2013 leveraged nearly complete sampling of *rbcL* of the ferns of Japan to conduct DNA barcoding surveys of gametophytes at multiple plots scattered throughout the country. 
Importantly, @Ebihara2013 developed new methods for surveying communities of unidentified fern gametophytes. 
This was not an issue in the papers described in the previous section, as those studies were focused on obtaining samples of a single population or a few populations of a single taxon. 
In order to quantify community-level diversity while maintaining sampling effort across sites, @Ebihara2013 developed a quadrat system as follows: preliminary surveys were first conducted to identify habitats harboring gametophytes, then a square 1 m^2^ quadrat subdivided into 25 10 x 10 cm squares was established using a garden net. 
A single gametophyte individual, if present, was sampled from the point nearest the center of each square of the net, then cut in half. 
One half was kept as a voucher and the other used for DNA barcoding. 
Using this method @Ebihara2013 showed that gametophytes with non-cordate morphologies, like those of the well-known North American independent gametophytes, tend to occur at large distances from sporophytes within the context of the Japanese fern flora. 
They also found three more taxa of independent gametophytes (two in *Hymenophyllum*, one in *Lomariopsis*) that lacked sporophyte counterparts in Japan. 

@Nitta2017 used the same quadrat sampling method as @Ebihara2013 to sample fern gametophytes and compare the community structure of gametophytes and sporophytes along an elevational gradient from ca. 200 m to 2,000 m on the islands of Moorea and Tahiti, French Polynesia. 
They used *rbcL* as a primary barcode marker with *trnH-psbA* as a secondary marker. 
Although *trnH-psbA* has lower discrimination power than other markers, it is easy to amplify, @Nitta2017 were able to use it in some cases when amplification of *rbcL* failed.
@Nitta2017 highlighted a challenge to this type of community survey: they observed far fewer species observed as gametophytes than sporophytes, and the distribution of gametophyte individuals was dominated by a small number of very common species, with a long tail of rare species. 
This is in contradiction to the expectation that there should be at least as many species in the gametophyte phase as sporophytes, and may be due to the skewed abundance distribution or seasonality in spore germination. 
To account for this in comparisons of fern gametophyte and sporophyte community composition, they used a reduced dataset only including species that had been observed at least once in each life stage across the gradient. 
This revealed that community structure does differ between fern sporophytes and gametophytes: sporophyte communities became increasingly phylogenetically clustered at high elevations, while gametophytes did not show any significant change in phylogeneti structure with elevation.

Indeed, this expectation of greater species richness in the gametophyte phase was borne out by the study of @Kuo2014a, who carried out exhaustive sampling of gametophytes throughout the year at a single site in Taiwan. 
They did this by establishing 12 subplots side-by-side at a single site in Taiwan and exhaustively sampling the subplots one per month, then comparing the observed diversity with the diversity of sporophytes in the plot. 
This demonstrates that careful sampling is needed to accurately compare community-level diversity between fern sporophytes and gametophytes.

* Community assembly in Tahiti [@Nitta2017]
* Japan gameto survey [@Ebihara2013]
* Taiwan gameto survey over year [@Kuo2014a]

**Other ecological studies**

DNA barcoding of fern gametophytes can have applications beyond simply identifying individuals to species. 
The morphology and ecological characteristics of the sampled individuals can then be used to study various aspects of their ecology once their identity is known. 
As with community ecology, such studies are still limited in number, but demonstrate the great potential for this approach.

@Nitta2009 used DNA barcoding to confirm the identity of gametophytes of *Vandenboschia collariata* (Bosch) Ebihara & Dubuisson, a hemi-epiphytic filmy fern from Costa Rica. 
Hemi-epiphytes are often poorly documented---typically, only a portion of the adult plant may be sampled from the host tree---so it is often unknown whether they are primary hemi-epiphytes, which start on the ground and climb up the tree, or secondary hemi-epiphytes, which start on the tree and send roots to the ground.
@Nitta2009 sampled gametophytes and observed juvenile sporophytes at the base of host trees, which confirmed the status of *V. collariata* as a primary hemi-epiphyte. 
The overall prevalence of primary vs. secondary hemi-epiphytism is still unknown for ferns generally. 
Since epiphytes are a major component of fern diversity, accounting for ca. 30% of extant species, and hemi-epiphytism was likely a involved in the evolution of epiphytism, the approach of @Nitta2009 could help provide insights into the transition from terrestrial to epiphytic growth in ferns.

@Ebihara2019a expanded on the results of @Ebihara2013 to conduct long-term monitoring of a population of *Pleurosoriopsis makinoi* (Maxim. ex Makino) Fomin gametophytes in Japan. 
The survey of @Ebihara2013 identified a population of *P. makinoi* gametophytes growing in a site without any sporophytes nearby. 
@Ebihara2019a then monitored growth of the population for 5 years and compared microclimate of the gametophyte population with that of a site supporting sporophytes. 
They showed that the gemmae of the gametophytes have seasonal periods of growth, growing in size and number during the spring and summer, and detaching during the winter. 
The microclimate analysis suggested that a possible cause of sporophyte suppression was low light in winter due to the planting of evergreen conifers. 
Long-term monitoring of other populations with individuals indetified by DNA barcoding would greately contribute to our understanding of population dynamics and recruitment in ferns.

Other studies have also built upon earlier DNA barcoding results to better understand fern ecology.
@Nitta2020 and @Nitta2021 both leveraged the DNA barcoding results of fern gametophytes on the islands of Moorea and Tahiti by @Nitta2017.
@Nitta2020 scored morphological traits including morphotype and presence of hairs, glands, and gemmae in the gametophytes surveyed by @Nitta2017, then including these trait data in a comparison of the functional community structure of terrestrial vs. epiphytic fern communities. 
They found that epiphytic fern communities were more functionally diverse than terrestrial communities, despite the generally harsher (drier) conditions of the canopy. 
Additional detailed study of fern gametophyte morphological and physiological traits with species identified by DNA barcoding could provide even greater insight into the processes governing community assembly of epiphytic and terrestrial fern communities. 

@Nitta2021 took a unique approach which we are not aware has been used by any other study: in order to compare adaptations to water stress between terrestrial and epiphytic filmy ferns, they conducted desiccation tolerance assays on gametophytes of unknown species at the field station where they conducted their field survey, then later identified the individuals to species using DNA barcoding in the lab (this was a subset of individuals included in the community diversity analysis of @Nitta2017). 
This resulted in rather skewed sampling numbers for some species (e.g., the most common species, *Callistopteris apiifolia* was sampled > 50 times, whereas rarer species were sampled only a few times or even just once each), but allowed them to sample a broad range of taxa without the need for cultivation of gametophytes from spores, which are often slow growing and may require > 1 year to attain enough material for physiological tests. 
One of the most surprising results of this study was that *C. apiifolia*, which had by far the most broadly distributed gametophytes yet sporophytes that are restricted to moist cloud forest > ca. 600 m, has gametophytes incapable of tolerating even slight amounts of desiccation. 
Rather, @Nitta2021 hypothesize that *C. apiifolia* gametophytes exploit small, well-protected microhabitats at the bases of rocks and trees to avoid desiccation and thus attain a broader range than conspecific sporophytes.
This points to the importance of microhabitat for understanding the distribution of fern gametophytes, which has also been supported by the finding of wide intraspecific variation in desiccation tolerance in vittarioid gametophytes mirroring variation in their environmental conditions [@Chambers2016; @Chambers2017].

* Growth rates and comparison of niche between G & S [@Ebihara2019a]
* Determining hemi-epiphyte status [@Nitta2009]
* Comparison of desiccation tolerance between filmy G & S [@Nitta2021]
* Community structure of epi vs. ter [@Nitta2020]

**Next-generation DNA barcoding**

* tissue direct PCR [@Li2009]
* next version of tissue direct PCR (Li-Yaung to send draft)
* multiplexing DNA barcoding [@Gostel2020]
* MinIon sequencing for field direct ID?
 
**CONCLUSIONS** 

**ACKNOWLEDGMENTS**

Li-Yaung Kuo kindly shared drafts of unpublished manuscripts.

**AUTHOR CONTRIBUTIONS** 

J.H.N. and S.M.C. wrote the manuscript.

**DATA AVAILABILITY**

Code to generate the mansucript is available at https://github.com/joelnitta/gameto-barcode-review.
 
**LITERATURE CITED**
